packages/middleware/src/apiClientFactory/applyContextToApi.ts

```diff
@@ -36,6 +38,17 @@ const applyContextToApi = <
         const extendQuery = createExtendQuery(context);
         const transformedArgs = await hooks.before({ callName, args });
         const apiClientContext = { ...context, extendQuery };
+        if (isCustomEndpointHandler(fn)) {
+          if (context.res.locals?.alokai?.metadata?.scope) {
+            context.res.locals.alokai.metadata.scope.extensionName =
+              fn._extensionName;
```

Comment:
If `fn` stands for function than it's weird to provide some property to it. Could you pls change the `fn` name to something more accurate?

Reply:
There is no better way to achieve this and in js fn is an object and can store properties - and it does in fact. I do no like that patter but considering the fact we are using such language and it will a BC if we will do it other way it is a lesser evil to me.